fera_full.py

```
@@ -1,3 +1,189 @@
+                gotoviewer = False
+                if(len(sys.argv) >= 3 and sys.argv[2]=='1'):
+                    gotoviewer = True  
+                sqliteconn = utilities_general.connectDB(str(global_settings.pathdb))
+                
+                utilities_general.gather_information_fromdb(sqliteconn)
+                sqliteconn.close()
+                utilities_general.initiate_indexing_thread()
+                if(not gotoviewer):
+                    global_settings.splash_window.window.withdraw()
+                    indexador_fera.App(global_settings.version, gotoviewer)
+                if(global_settings.pathdb==None):
```

Ruff

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)